cabal.project

```diff
@@ -3,3 +3,7 @@ packages:
   ./plutarch-*
 
 benchmarks: true
+
+-- Enabling the flag here allows TraceSpec.hs and such tests to actually test
+-- the `ptrace` family of functions.
+flags: -development
```

Comment:
This doesn't change existing behaviour. `development` remains turned off by default.

When doing #191 however we should figure out a way to toggle this at project level in flake.nix checks.